wetterdienst/util/cache.py

```diff
@@ -12,14 +12,18 @@
 log = logging.getLogger()
 
 # Python on Windows has no "fcntl", which is required by the dbm backend.
-# TODO: Make backend configurable, e.g. better use Redis.
+
 platform = platform.system()
 backend = "dogpile.cache.dbm"
 if platform == "Windows":
     backend = "dogpile.cache.memory"
 
 # Compute cache directory.
-cache_dir = appdirs.user_cache_dir(appname="wetterdienst")
+try:
+    cache_dir = os.environ['CACHE_DIR']
```

Comment:
Please use "WD_CACHE_DIR" instead. I think the variable name "CACHE_DIR" is to general.

Reply:
Done